td_core/mdpyget/bots/to_sql.py

```
@@ -93,6 +97,33 @@ def update_table(list_of_lines, table_name, columns, lento=10, title_column="tit
         print(f"to_sql.py update_table() {done} done, from {len(list_of_lines)}.")
 
 
+def update_table_2(list_of_lines, table_name, columns_to_set=[], lento=10, columns_where=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)